editor/js/Editor.js

```diff
@@ -131,8 +131,7 @@ Editor.prototype = {
 		this.scene.uuid = scene.uuid;
 		this.scene.name = scene.name;
 
-		this.scene.background = ( scene.background !== null ) ? scene.background.clone() : null;
```

Comment:
I can't remember why we were cloning `scene.background`.

Reply:
Maybe because `scene.fog` is cloned, too? 

It seems the line was added like that right from the beginning: https://github.com/mrdoob/three.js/commit/750973beb1e86f4a5a89b78637eab58bec146017#diff-c7a0a223e819b2c4e36644546146290d9d14f241cf78daa5438faf335a2aff35R124

Reply:
Since you have changed the code, it makes sense to align:

https://github.com/mrdoob/three.js/blob/8d30d742e3d719fb090f811ace5e5d5fc863a5b1/editor/js/Editor.js#L136

Reply:
Done! https://github.com/mrdoob/three.js/commit/64c69901c01ea8b7ca0f7a0b54d3d950d72a9331